java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/EndpointHostResolver.java

```diff
@@ -117,14 +117,15 @@ synchronized void resolve(
     }
 
     synchronized void destroy() {
-        assert (!_destroyed);
```

Comment:
The interrupt test was hitting this assert when calling destroy after an interrupted communicator destroy.